lib/drivers/medtronicDriver.js

```diff
@@ -481,19 +486,25 @@ module.exports = function (config) {
 
                     } else {
                       var decoded = decodeMessage(packet.parser, result.bytes);
-                      page.set(decoded.message, length);
-                      length += decoded.message.length;
-                      console.log('Part', count, 'of page (length: ', decoded.message.length ,'):', common.bytes2hex(decoded.message));
-
-                      var payload = _.cloneDeep(result.bytes);
-                      payload[32] = 0x00; // mask out checksum itself
-                      var checksum = result.bytes[32];
-                      debug('Calculated checksum:', _sum_lsb(payload));
-                      debug('Checksum from packet:', checksum);
-                      if( (_sum_lsb(payload) !== checksum) || (decoded.message.length === 0)) {
-                        debug('Invalid checksum or null data');
-                        return callback(new Error(COMM_ERROR_TEXT));
-                      };
+
+                      if(decoded.message.length === 2 && decoded.message[0] === ASCII_CONTROL.NAK) {
```

Comment:
`decoded` will have a `message` field if it's an `instanceof Error` (as has been added above), but the behaviour may not be what you think...